js/views/settingsVw.js

```diff
@@ -171,16 +184,109 @@ module.exports = Backbone.View.extend({
     });
     return this;
   },
-/* this is not how this should work
-  renderBlocked: function (model) {
-    "use strict";
-    this.blockedList = new personListView({model: model,
-                                           el: '.js-list1',
-                                           title: "No one blocked",
-                                           message: ""});
-    this.subViews.push(this.blockedList);
+
+  patchAndFetchBlockedUsers: function(models) {
+    var self = this;
+
+    if (models && models.length) {
+      __.each(models, function(model) {
+        model.urlRoot = self.serverUrl + 'profile';
+        
+        // Monkey patching parse so the profile is not nested
+        // and therefore change events will be in play.
+        model.oldParse = model.parse;
+        model.parse = function (response) {
+          if (response.profile) {
+            return model.oldParse(response).profile;
+          } else {
+            return response;
+          }          
+        };
+
+        model.fetch({ data: { guid: model.get('guid')} });
+      });
+    }
+  },
+
+  renderBlocked: function(options) {
+    var self = this,
+        modelsPerBatch = 25,
```

Comment:
If you want to test the lazy load functionality without blocking a massive amount of user, change the modelsPerBatch value to something low, like 2. If you really want to step the test up a level, add one of the min height classes to the BlockedUserVw (in the classname on the view file).